### DISCUSSION TITLE
about ibm_zos_core.zos_job_submit / iconv error

### DISCUSSION THREAD

**Discussion by Emi-Furubayashi** [ansible-collections/ibm_zos_core#1206] [Q&A]
When using module zos_job_submit, the following error occurs in the case of specifying location=LOCAL , encode from UTF-8 to IBM-1047.
 The input file (UTF-8 , LF text JCL file) has been tested with other methods of iconv (at ubuntu , or , host with BPXBATCH) and confirmed that there's no error, the converted text JCL file are executed successfully at host.
Please let me know how to deal with it or if there are any cases with the same error.
zos-Python ver is 3.11.X.
ansible-Python ver is 3.8.16
collections ver is 1.4
zos ver is 2.5
zoau ver is unknown

   "stderr": "iconv: 1 illegal character sequence(s) for codeset in input file\\n",
    "stderr_lines": [
        "iconv: 1 illegal character sequence(s) for codeset in input file"

**Comment by ddimatos** [comment]
Hello @Emi-Furubayashi , this appears to be a duplicate to the issue you opened #1211 , since the git issue has more content we can discuss there there, thus I will close this and mark it as a duplicate.
